src/extension/background-script/connectors/alby.ts

```diff
@@ -224,6 +228,7 @@ export default class Alby implements Connector {
         client_id: clientId,
         client_secret: clientSecret,
         callback: redirectURL,
+        user_agent: "lightning-browser-extension",
```

Comment:
@bumi could you please confirm this is the correct user agent?

Reply:
maybe we also add the version there. (which we could set in an environment variable / load from the package.json) 
https://github.com/getAlby/lightning-browser-extension/blob/7139fead6b9603ad9998cc4f28b5a15fd70ff401/webpack.config.js#L33-L35

Reply:
resolved